diff --git a/node_modules/@wext/shipit/lib/firefox.js b/node_modules/@wext/shipit/lib/firefox.js
index 96bceac..ae895fb 100644
--- a/node_modules/@wext/shipit/lib/firefox.js
+++ b/node_modules/@wext/shipit/lib/firefox.js
@@ -8,6 +8,7 @@ const webextBinPath = require.resolve('.bin/web-ext')
 // https://github.com/Originate/node-text-stream-search/issues/31
 let StreamSearch = require('text-stream-search')
 if (StreamSearch.default) StreamSearch = StreamSearch.default
+if (StreamSearch.TextStreamSearch) StreamSearch = StreamSearch.TextStreamSearch;
 
 module.exports = async function (source) {
   let errors = []
